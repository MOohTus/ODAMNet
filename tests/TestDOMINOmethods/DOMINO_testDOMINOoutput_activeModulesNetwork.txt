source	target	link	AMI_number
g2	g5	PPI	1
g2	g4	PPI	1
g2	g1	PPI	1
g4	g1	PPI	1
g4	g20	PPI	1
g7	g6	PPI	3
g7	g16	PPI	3
g7	g17	PPI	3
g16	g6	PPI	3
g16	g17	PPI	3
g9	g14	PPI	2
g9	g13	PPI	2
g9	g18	PPI	2
g18	g14	PPI	2
