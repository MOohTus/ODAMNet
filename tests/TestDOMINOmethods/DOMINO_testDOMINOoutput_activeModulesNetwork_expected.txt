source	target	link	AMI_number
g5	g2	PPI	1
g2	g4      PPI     1
g2	g1      PPI     1
g1	g4      PPI     1
g4	g20      PPI     1
g6	g7	PPI	3
g16	g7      PPI     3
g16	g17      PPI     3
g7	g17      PPI     3
g9	g14	PPI	2
g9	g13     PPI     2
g9	g18     PPI     2
g14	g18     PPI     2
